x-pack/plugins/observability/public/pages/slo_edit/helpers/process_slo_form_values.ts

```diff
@@ -133,7 +133,7 @@ function transformPartialIndicatorState(
         params: Object.assign({}, CUSTOM_METRIC_DEFAULT_VALUES.params, indicator.params ?? {}),
       };
     default:
-      assertNever(indicatorType);
+      assertNever(indicatorType as never);
```

Comment:
We need to handle the `case 'sli.metric.timeslice'`: above and remove the `as never` here. 


We basically take the default custom timeslice indicator values, and merge them with whatever is provided in the `_a` url query parameter. This way we allow partial indicator to be provided, without breaking the form. This is partially refactored in this opened PR https://github.com/elastic/kibana/pull/168760/files#diff-c8912979b17b75623a8afe8fa88f89022ae1fca1fb4e01c9dc89438d16206166L140